app/views/datasets/show.html.erb

```diff
@@ -19,7 +19,7 @@
       <%= render "supporting_docs" %>
   <% end %>
 
-  <% if @dataset['organisation']['contact_name'] || @dataset['organisation']['contact_email'] %>
+  <% if @dataset['organisation']['contact_email'].present? %>
```

Comment:
Yes agree. This is just a first pass on the back of initial discussion with Jonathon. Acceptance criteria here - https://trello.com/c/aOhd5nFJ/332-test-cases-dataset-view-page